src/login/loginctl.c

```diff
@@ -282,14 +282,15 @@ static int list_seats(int argc, char *argv[], void *userdata) {
 
         for (;;) {
                 const char *seat;
+                TableCell *cell;
 
                 r = sd_bus_message_read(reply, "(so)", &seat, NULL);
                 if (r < 0)
                         return bus_log_parse_error(r);
                 if (r == 0)
                         break;
 
-                r = table_add_cell(table, NULL, TABLE_STRING, seat, arg_full);
+                r = table_add_cell(table, &cell, TABLE_STRING, seat, arg_full);
                 if (r < 0)
```

Comment:
Why is this change needed?

Reply:
Right, I removed the unnecessary cell variables.